Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION}
+
+WORKDIR /vulnpy
+COPY . .
+
+ARG FRAMEWORK="wsgi"
+
+RUN pip install -U contrast-agent
```

Comment:
I'm considering decoupling agent installation from this Dockerfile in the next iteration. This would give us a nice base image that doesn't require rebuilding the agent each time. For a first pass, this was easier to wrap my head around